sql/driver/doc.go

```diff
@@ -235,6 +238,45 @@ Querying rows:
 
 	SELECT __key, name FROM person WHERE age > 30
 
+2) "json" is a first-class SQL type with IMap and Kafka Connector support.
+
+Creating a mapping:
+
+	CREATE MAPPING person (
+		__key BIGINT,
+		this  JSON
+	)
+	TYPE IMAP
+	OPTIONS (
+		'keyFormat' = 'bigint',
+		'valueFormat' = 'json'
+	)
+
+Inserting rows:
+
+	// Use serialization.JSON type to specify a JSON value.
+	INSERT INTO person VALUES(100, serialization.JSON(fmt.Sprintf(`{"age":%d, "name":%s}`, 35, 'Jane Doe')))
+
+Querying rows:
+
+	// Use serialization.JSON type to scan JSON string.
+	q := fmt.Sprintf(`SELECT this FROM "%s" WHERE CAST(JSON_VALUE(this, '$.age') AS DOUBLE) > ?`, mapName)
+	rows, err := db.Query(q, minAge)
+	for rows.Next() {
+		var js serialization.JSON
+		rows.Scan(&js)
+	}
+
+Supported JSON related operations:
```

Comment:
Could you put a link to the main Hazelcast documentation?

Reply:
There isn't one yet 😔

Reply:
There is a link for the beta at: https://docs.hazelcast.com/hazelcast/5.1-beta-1/sql/working-with-json 5.1 link will probably be: https://docs.hazelcast.com/hazelcast/5.1/sql/working-with-json